extrinsic_matrix:
  - [-0.023901636699062705938,  0.99962208920580499905,  0.013578589237173755557, -0.0089014391076933259688]
  - [-0.2549996728945386421,  0.007037093575383479685, -0.96691558329612305637, -0.32459588152429603004]
  - [-0.96664568586181246864, -0.026573462579366717832,  0.25473504470966757, -0.053184347194154764765]
  - [ 0.0000,  0.0000,  0.0000,  1.0000]